website/docs/metrics.md

```diff
@@ -1,11 +1,87 @@
 ---
 id: metrics
-title: Metrics
+title: Metrics & Observability
 ---
+## Observability
+
+We want to enable operators to glean information about how Gatekeeper is performing, as a system. To that end, operators can choose to enable execution stats that are gathered when Constraints are evaluated in the engine. At present these stats can only be logged.
+
+### Logging Constraint Execution Stats
+
+- set `--log-stats-audit`. This flag enables logging the stats for the audit process.
+
+- set `--log-stats-admission`. This flag enables logging the stats for the admission review process.
+
+By default, these new logging features are disabled due to their high-volume data generation. 
```

Comment:
No need to reference the justification here. There are multiple reasons why this is disabled by default (for instance: not incurring a potential performance hit).

We probably should have a caveats section that does list log volume as a downside.